### PR TITLE
Task #48972 [User] Đăng nhập

### PR DESCRIPTION
**Pull request by anhlh-3049**
## Related Tickets
- https://edu-redmine.sun-asterisk.vn/issues/48972

## WHAT (optional)
- Login 

## Evidence (Screenshot or Video)
Screenshot_3

### CODE REVIEW

**Comment by anhlh-3049** [comment]
ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  end
+
+  def current_user
+    return unless session[:user_id]
+
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    !current_user.nil?
```
Comment (ghost) [inline app/helpers/sessions_helper.rb]:
```suggestion
    current_user.present?
```

Comment (ghost) [inline app/helpers/sessions_helper.rb]:
```suggestion
    current_user.present?
```

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    </div>
+      <div class="col-md-12">
+        <div class="login-form">
+          <%= form_for :session, url: login_path do |f| %>
+
+          <%= f.label :email %>
+          <%= f.email_field :email, class: "form-control" %>
+
+          <%= f.label :password %>
+          <%= link_to "t(".forgot_password")" %>
```
Comment (ghost) [inline app/views/sessions/new.html.erb]:
```suggestion
          <%= link_to t(".forgot_password"), "#" %>
```

Comment (ghost) [inline app/views/sessions/new.html.erb]:
```suggestion
          <%= link_to t(".forgot_password"), "#" %>
```

**Review by ghost — COMMENTED** [review]
cmt

**Comment by anhlh-3049** [comment]
ready

File: config/locales/en.yml [hunk 3]
```diff
@@ -23,5 +23,18 @@ en:
...
+    new:
+      provide_title: "Log in"
+      title: "Login"
+      service_html: "Please fill in the information below to log in"
+      forgot_password: "Forgot Password"
+      remember: "Remember me on this computer"
+      btn_title: "Log in"
+    create:
+      flash_danger: "Invalid email/password combination"
+
```
Comment (ngocvt-0484) [inline config/locales/en.yml]: thừa dòng

**Pull request merged**